bip-undef-0.mediawiki

```diff
@@ -0,0 +1,113 @@
+<pre>
+  BIP: ???
+  Title: Peer Authentication
+  Author: Jonas Schnelli <dev@jonasschnelli.ch>
+  Status: Draft
+  Type: Standards Track
+  Created: 2016-03-23
+</pre>
+
+== Abstract ==
+
+This BIP describes a way that peers can authenticate to other peers to guarantee ownership and/or allowing to access additional or limited services.
```

Comment:
Suggestion:
This BIP describes a secure authentication protocol for use between peers within the Bitcoin network.